.github/ISSUE_TEMPLATE.md

```diff
@@ -1,10 +1,13 @@
 ## Description
-> Describe the issue you've observed
+> **IMPORTANT**: if the defect is reproduced only in a workflow from within the Visual Studio IDE then do not report the issue here - instead, please report it using Visual Studio's "Send Feedback" option that can be accessed from the Help menu.
```

Comment:
May be add a link to https://developercommunity.visualstudio.com/spaces/8/index.html